cluster_library.c

```diff
@@ -2084,23 +2099,25 @@ PHP_REDIS_API void cluster_gen_mbulk_resp(INTERNAL_FUNCTION_PARAMETERS,
 {
     zval z_result;
 
-    /* Return FALSE if we didn't get a multi-bulk response */
-    if (c->reply_type != TYPE_MULTIBULK) {
+    /* Abort if the reply isn't MULTIBULK or has an invalid length */
+    if (c->reply_type != TYPE_MULTIBULK || c->reply_len < -1) {
```

Comment:
`c->reply_len < 0`

Reply:
I think this one is correct.  `*-1\r\n` is a valid NULL multi bulk.

Reply:
After loading next lines I see that this condition is correct